### DISCUSSION TITLE
Migrate Existing Wordpress Site with subdomain for images and attachments

### DISCUSSION THREAD

**Discussion by EdenSpire** [EasyEngine/easyengine#1779] [Support Requests]
Hello,

New to EasyEngine World.
Hopefully I can contribute to this community.

At the moment, I want to move my site from a limited shared hosting to EasyEngine.
The problem is, my site is separated into two, domain.tld for main files, and subdomain.domain.tld for images/attachment/except JS and CSS.

How to migrate this kind of site to EasyEngine?

Any help is highly appreciated.

Thank you.

**Comment by marcoandrei** [comment]
@EdenSpire

You have to create 2 sites with the domains you need. 

The first appears to be the WP installation, so you can create the site with `--type=wp` option.

The second one appears to be a plain HTML site, use `-–type=html` option to create it.

Check the documentation: https://easyengine.io/commands/site/create/

Good luck! 
Saúde!
